packages/protobuf/package.json

```diff
@@ -24,6 +24,12 @@
       "module": "./dist/esm/index.js",
       "require": "./dist/cjs/index.js",
       "import": "./dist/proxy/index.js"
+    },
+    "./dist/esm/private/*": {
+      "module": "./dist/esm/private/*"
+    },
+    "./dist/cjs/private/*": {
+      "require": "./dist/cjs/private/*"
```

Comment:
Though technically only the second export is required to satisfy typescript, I added the esm export for completeness.